lib/models/application.coffee

```diff
@@ -149,6 +149,53 @@ getApplicationModel = (deps, opts) ->
 		.asCallback(callback)
 
 	###*
+	# @summary Get a single application using the appname and owner's username
+	# @name getAppWithOwner
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String} appName - application name
+	# @param {String} owner - The owner's username
+	# @param {Object} [options={}] - extra pine options to use
+	# @fulfil {Object} - application
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.getAppWithOwner('MyApp', 'MyUser').then(function(application) {
+	#		console.log(application);
+	# });
+	###
+	exports.getAppWithOwner = (appName, owner, options = {}, callback) ->
+		callback = findCallback(arguments)
+
+		a.pine.get
```

Comment:
Oh shoot. I've copied the debugging code out of another file. It's essentially the same, but with different vars... Sorry for the noise.